### PR TITLE
docs: correct manual page links for version 3.0

### PR DESCRIPTION
**Pull request by mspncp**
_(For consistency reasons; inspired by #17849)_

### CODE REVIEW

**Review by t8m — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ Manual Pages
 The manual pages for the master branch and all current stable releases are
 available online.
 
-- [OpenSSL master](https://www.openssl.org/docs/manmaster)
+- [OpenSSL master](https://www.openssl.org/docs/man3.0)
```
Comment (t8m) [inline README.md:128]: Oops, this is wrong - should you add the 3.0 link instead?

Comment (mspncp) [reply README.md:128]: Ah, good catch!  My fault, it's the result of a recursive sed replace, and I didn't notice it when looking at the diff.

Comment (mspncp) [reply README.md]: done.

**Review by levitte — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (t8m) [reply README.md]: Btw, this should be done in master as well, shouldn't it? Would you open a separate PR?

Comment (mspncp) [reply README.md]: Done, see #17855.

**Review by paulidale — APPROVED** [review]

**Comment by mspncp** [comment]
Merged to 3.0 in f967cdbfb9, thank you!

**Pull request closed without merging**